common/dynamicconfig/constants.go

```diff
@@ -84,6 +84,11 @@ var (
 		9000,
 		`VisibilityPersistenceMaxWriteQPS is the max QPC system host can query visibility DB for write.`,
 	)
+	VisibilityPersistenceSlowQueryThreshold = NewGlobalDurationSetting(
+		"system.visibilityPersistenceSlowQueryThreshold",
+		time.Second,
```

Comment:
Historically, we have been checking with 2 seconds threshold.

Reply:
I choose the threshold after a quick look at metrics.

From the PR Description
>Risks Threshold is too low, causing excessive logging. Existing metrics for Query Latency were evaluated in largest cells to check f or rate. Expect >1 log per hour, per cell.

I think a 1 sec will create very minimal amounts of new logs, while providing us with better visibility into long running queries.

Is there something better about setting the threshold at 2 secs?